test/modelnode.jl

```diff
@@ -334,7 +334,35 @@ end
                   ]
             m = reflectinmodel(ds, layerbuilder, abuilder)
             @inferred m(ds)
-            @test gradtest(() -> m(ds).data, Flux.params(m))
+            @test gradtest(() -> m(ds).data, params(m))
         end
     end
 end
+
+@testset "testing simple named tuple model with reduce catobs in gradient" begin
+    layerbuilder(k) = Dense(k, 2, relu)
+    x = ProductNode((node1 = BagNode(ArrayNode(randn(Float32, 3, 4)), [1:2, 3:4]),
+                     node2 = BagNode(ArrayNode(randn(Float32, 4, 4)), [1:1, 2:4])))
+    m = reflectinmodel(x, layerbuilder)
+    ps = params(m)
+    @test gradient(() -> sum(m([x, x]).data), ps) isa Grads
```

Comment:
What is the point of this line?

Reply:
The point is to test if the gradient is not nothing.
Do you think it would be better to test if it's not nothing than that it's Grads?